tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -136,3 +138,30 @@ def test_hook_property(get_clients) -> None:
 
     assert (rst.T.shares[0] == x.T).all()
     assert (rst.T.shares[1] == y.T).all()
+
+
+@pytest.mark.parametrize("parties", [2, 3, 5, 7, 11])
```

Comment:
Hrishi, I think we would include a sanity check in the future which prevents initializing falcon with other than `3`  parties. This test scenario would conflict that change.